src/lib.rs

```diff
@@ -190,6 +211,7 @@ pub fn tx_origin() -> [u8; 20] {
     ret
 }
 
+/// Appends log data to the transaction receipt, with a variable number of topics.
```

Comment:
Does this needs a documentation if it is not public? Will be included in the external documentation?